### PR TITLE
1288 Update Spinner SVG, Renamed to Progress Spinner

### LINKED ISSUE #1288: Spinner: update to DS v3.1.1 and rename to progress-spinner
Full details & specs can be found in internal Figma/playbook.

I think we might also rename this to progress-spinner, in line with progress-bar and progress-stepper.

Screen Shot 2020-11-06 at 12 10 52 PM

### PR DESCRIPTION
**Pull request by austingardner**
## Description
Updated Spinner SVG with new assets.
Renamed to Progress Spinner

## References
Closes #1288 

## Screenshots
Screen Shot 2021-02-16 at 3 53 59 PM

### CODE REVIEW

File: dist/variables/ds4/spinner-variables.less [hunk 1]
```diff
@@ -1 +1,2 @@
 @spinner-image-url: "https://ir.ebaystatic.com/cr/v/c1/skin/svg/spinner/v1/ds4/spinner.svg";
```
Comment (austingardner) [inline dist/variables/ds4/spinner-variables.less:1]: Should I delete these from the dist, or is there a file that tells the dist not to generate these files?

File: src/less/progress-spinner/base/progress-spinner.less [hunk 2]
```diff
@@ -20,6 +20,17 @@
 }
 
 .spinner--large {
+    background-image: url(@spinner-large-image-url);
     height: 60px;
     width: 60px;
 }
+
+@media (prefers-color-scheme: dark) {
```
Comment (austingardner) [inline src/less/progress-spinner/base/progress-spinner.less:28]: I wasn't sure how we wanted to manage dark mode with the progress spinner, let me know what I should do here.

As is this works, but I'm not sure if that's how we're managing things at present.

**Review by ianmcburnie — CHANGES_REQUESTED** [review]
For backwards compatibility. we'll need to leave in all file and class references to the name of `spinner`. On the website and in storybook we should only show the new `progress-spinner` version. When we get to the next major version we can delete all references to the old name.

**Comment by austingardner** [comment]
Okay thanks for the guidance!

File: dist/variables/ds4/spinner-variables.less (hunk 1 above)
Comment (ianmcburnie) [reply dist/variables/ds4/spinner-variables.less:1]: I'm pretty sure we should still be keeping files in dist under version control. @agliga can confirm.

File: src/less/progress-spinner/base/progress-spinner.less (hunk 2 above)
Comment (ianmcburnie) [reply src/less/progress-spinner/base/progress-spinner.less:28]: Look at any of the ds6 properties files. That's where we manage dark mode.

Example: https://github.com/eBay/skin/blob/master/src/less/properties/ds6/switch-properties.less

**Pull request closed without merging**